apps/pigment-css-next-app/package.json

```diff
@@ -12,21 +12,20 @@
     "@pigment-css/react": "^0.0.16",
     "@mui/utils": "workspace:^",
     "@mui/base": "workspace:^",
+    "@mui/icons-material": "workspace:^",
     "@mui/lab": "workspace:^",
     "@mui/material": "workspace:^",
-    "@mui/system": "workspace:^",
     "@mui/material-nextjs": "workspace:^",
-    "@mui/icons-material": "workspace:^",
-    "@emotion/cache": "latest",
```

Comment:
Is `@emotion/cache` no longer needed?

Reply:
Not sure what happened, probably a bad merge. I think it's not used though, but that's for another PR. Fixed.